### PR TITLE
Added workflows for collision 22 (900GeV) runs

### PR DESCRIPTION
**Pull request by pkalbhor**

### CODE REVIEW

File: alcaval_steps.py [hunk 1]
```diff
@@ -116,3 +128,27 @@
...
+steps['RunMinimumBias2022A_v2']={'INPUT':InputInfo(dataSet='/MinimumBias/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+
+steps['HLT_Collision22_v1'] = merge( [ {
+                '-s': 'L1REPACK:uGT,HLT',
+                '--customise': 'L1Trigger/Configuration/customiseUtils.L1TGlobalMenuXML'
+                }, step2Defaults] )
+steps['RECO_Collision22_v1']=merge([{
+                            '-s'            : 'RAW2DIGI,L1Reco,RECO,DQM'},
+                            {'--procModifiers'  : 'siPixelQualityRawToDigi'},
+                            {'--customise':'Configuration/DataProcessing/RecoTLR.customisePostEra_Run3,RecoLocalCalo/Configuration/customiseHBHEreco.hbheUseM0FullRangePhase1'},
```
Comment (pkalbhor) [inline alcaval_steps.py:152]: @tvami , all,
Please leave comments if you think any changes to current configuration is required.

Comment (tvami) [reply alcaval_steps.py:152]: I think it should be good, but I'm uncertain about `hbheUseM0FullRangePhase1` I'll ask HCAL

Comment (pkalbhor) [reply alcaval_steps.py:152]: I will keep this configuration until then

File: alcaval_steps.py [hunk 2]
```diff
@@ -116,3 +128,27 @@
...
+steps['RunZeroBias2022A_v2']={'INPUT':InputInfo(dataSet='/ZeroBias/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+steps['RunMinimumBias2022A_v2']={'INPUT':InputInfo(dataSet='/MinimumBias/Run2022A-v1/RAW',ls={353060: [[1,500]]})}
+
+steps['HLT_Collision22_v1'] = merge( [ {
+                '-s': 'L1REPACK:uGT,HLT',
+                '--customise': 'L1Trigger/Configuration/customiseUtils.L1TGlobalMenuXML'
+                }, step2Defaults] )
+steps['RECO_Collision22_v1']=merge([{
+                            '-s'            : 'RAW2DIGI,L1Reco,RECO,DQM'},
+                            {'--procModifiers'  : 'siPixelQualityRawToDigi'},
```
Comment (mmusich) [inline alcaval_steps.py:151]: @pkalbhor @tvami FYI this is not needed anymore (and can be actually detrimental).

Comment (pkalbhor) [reply alcaval_steps.py:151]: thanks @mmusich
 `'--procModifiers'  : 'siPixelQualityRawToDigi'` this one is not needed right?

Comment (mmusich) [reply alcaval_steps.py:151]: right.

**Pull request merged**